pkgs/applications/editors/atom/default.nix

```diff
@@ -54,7 +54,7 @@ let
 
     preFixup = ''
       gappsWrapperArgs+=(
-        --prefix "PATH" : "${gvfs}/bin"
+        --prefix "PATH" : "${glib}/bin"
```

Comment:
```suggestion
        --prefix "PATH" : "${glib.bin}/bin"
```

just to be certain. and can a comment also be added? (for gio)

Reply:
Something like this?

Reply:
yep